docs/product/relay/operating-guidelines.mdx

```diff
@@ -31,6 +31,21 @@ The following recommendations assume that Relay is run in Docker.
 
 </p>
 
+## Relay Binary
+
+Every release of Relay also provides a set of binaries that come with a minimum required OS version.
+
+| Operating System | Version                   |
+|------------------|---------------------------|
+| Ubuntu x86       | 20.04                     |
+| Ubuntu ARM       | 22.04                     |
+| MacOS            | macOS Sonoma (version 14) |
+| Windows          | Windows Server 2019       |
```

Comment:
How do we guarantee MacOS and WIndows?

Reply:
> We do have runners for those, or do you mean what would happen if Github discontinues those before the offical EOL?

Correct, how do we guarantee what you documented here, it's the same with ubuntu, github turned off the runners before the actual EOL. So if we promise it here, how do we make sure we can hold that promise.